src/main/java/org/broadinstitute/hellbender/cmdline/CommandLineParser.java

```diff
@@ -513,10 +649,22 @@ private String makeArgumentDescription(final ArgumentDefinition argumentDefiniti
         return sb.toString();
     }
 
+    private void usageForPluginDescriptorArgument(final ArgumentDefinition argDef, final StringBuilder sb) {
+        final GATKCommandLinePluginDescriptor<?> descriptor = (GATKCommandLinePluginDescriptor<?>) argDef.parent;
+        // this argument came from a plugin descriptor; delegate to get the list of allowed values
+        final Set<String> allowedValues = descriptor.getAllowedValuesForDescriptorArgument(argDef.getLongName());
+        if (allowedValues == null) {
```

Comment:
Usually we'd want to prefer empty collections over null values, is there a need to distinguish the two here